arangod/Cluster/ClusterInfo.cpp

```diff
@@ -783,6 +785,73 @@ void ClusterInfo::loadPlan() {
     }
   }
 
+  // "Plan":{"Analyzers": {
+  //  "_system": {
+  //    "Revision": 0,
+  //    "BuildingRevision": 0
+  //  },...
+  // }}
+
+  // Now the same for analyzers:
+  auto planAnalyzersSlice = planSlice.get("Analyzers");  // format above
+
+  if (planAnalyzersSlice.isObject()) {
+    swapAnalyzers = true;  // mark for swap even if no databases present to ensure dangling datasources are removed
+
+    for (auto const databasePairSlice : velocypack::ObjectIterator(planAnalyzersSlice)) {
```

Comment:
"databasePairSlice" is not a very good name as there could will be 2 more memebers - coordinator and rebootid. Maybe we should name it smth like  "databaseDataSlice"